chain/chain/src/resharding/types.rs

```diff
@@ -0,0 +1,13 @@
+use crate::flat_storage_resharder::FlatStorageResharder;
+use near_async::messaging::Sender;
+
+#[derive(actix::Message, Clone, Debug)]
+#[rtype(result = "()")]
+pub struct FlatStorageSplitShardRequest {
+    pub resharder: FlatStorageResharder,
+}
+
+#[derive(Clone, near_async::MultiSend, near_async::MultiSenderFrom)]
+pub struct ReshardingSender {
+    pub flat_storage_split_shard_send: Sender<FlatStorageSplitShardRequest>,
```

Comment:
In the future we'd want a sender for any type of resharding request and SplitShard would be one of the variants in the Request enum. No need to do it now though.